wiki/Visual_Settings/en.md

```diff
@@ -1,19 +1,19 @@
 
-Alternatively, it is possible to access Visual Settings overlay by pausing the game. However, this method does not work in [Multi](/wiki/Multi "Multi") match as it will be treated as attempting to quit the match and leave the room instead.
+Alternatively, it is possible to access visual settings overlay by pausing the game. However, this method does not work in [Multi](/wiki/Multi) matches as it will be treated as attempting to quit the match and leave the room instead.
 
-_Note:_ Setting changes from Visual Settings are saved per beatmap and will be lost after closing osu!. Global settings can be set in the [Options](/wiki/Options "Options") sidebar.
+Notice: Visual settings changes are saved per beatmap but will be lost after closing osu!. To set these globally (and forever) use the [options](/wiki/options) sidebar.
 
-## Available options
+## Settings
 
 | Name | Effect | Notes |
-|:---:|:---|:---|
-| Background dim | Darken the playfield. | During breaks, the dim is -30% of the rate set or 0%. No special effects at 100% dim. Also darkens storyboard elements at 100% dim. |
-| Disable storyboard | Remove all storyboard elements. Does not affect Kiai Time and background video. | In maps with an epilepsy warning, this is recommended for players with epilepsy issue. This option is disabled if there is no storyboard to play. |
-| Ignore beatmap skin | Do not use the beatmap's custom skin, if any. The player's selected skin will be used instead. | Requires a retry to take effect. |
-| Ignore beatmap hitsounds | Do not use the beatmap's custom hit sounds, if any. The player's selected skin's hit sounds will be used. | Requires a retry to take effect. |
-| Disable video | Do not play the video. This does not remove the storyboard. | Requires a retry if activated after the song starts. This option is disabled if there is no video to play. |
\ No newline at end of file
+| --- | --- | --- |
+| `Background dim` | Darken the playfield (including storyboards). | During breaks, the dim is decreased by 30% (max 0%). This effect can be disabled in the options. |
+| `Disable storyboard` | Remove all storyboard elements. This does not affect [Kiai Time](/wiki/Kiai_Time) and the background video, if any. | This is recommended for players with epilepsy issues for when the beatmap displays an epilepsy warning. This option is disabled if there is no storyboard to play. |
+| `Ignore beatmap skin` | Use the player's selected skin instead of the beatmap's included skin. | This requires a retry to take effect. |
+| `Ignore beatmap hitsounds` | Use the the player's selected skin's hit sounds instead of the beatmap's custom hit sounds, if any. | Requires a retry to take effect. |
+| `Disable video` | Do not play the background video. This does not remove the storyboard. | Requires a retry if activated after the gameplay begins. This option is disabled if there is no background video to play. |
```

Comment:
`Requires` -> `This requires`

For consistency from the one you edited from the `Ignore beatmap skin` notes.